waspc/src/Wasp/Generator/WebAppGenerator/Common.hs

```diff
@@ -89,3 +92,6 @@ mkTmplFdWithDstAndData srcPathInWebAppTemplatesDir dstPathInWebAppRootDir tmplDa
     (webAppRootDirInProjectRootDir </> dstPathInWebAppRootDir)
     (webAppTemplatesDirInTemplatesDir </> srcPathInWebAppTemplatesDir)
     tmplData
+
+getOnAuthSucceededRedirectToOrDefault :: AS.App.Auth.Auth -> String
+getOnAuthSucceededRedirectToOrDefault auth = fromMaybe "/" (AS.App.Auth.onAuthSucceededRedirectTo auth)
```

Comment:
I'd move this out of `Common` and into `AuthG` since its auth-specific.

Reply:
Good point. I was thinking this may be a common enough need to make more general (pun intended) but we can cross that bridge as necessary (or just export from `AuthG` in future).